### ISSUE TITLE
asking about sunscreen at the wrong time, leads to crash

### ISSUE THREAD

**Issue by ulno** [Daenara/rhasspy_weather#5]
Asking if you need sunscreen today, when it is already evening, creates this error (seems like the forecast becomes empty for the sunscreen-relevant time).

```
  File "cli_weather.py", line 50, in get_weather_forecast
    response = WeatherReport(request, forecast).generate_report()
  File "/home/ulno/sync/work/devel/cli_weather/rhasspy_weather/data_types/report.py", line 86, in generate_report
    response = self.__generate_item_report()
  File "/home/ulno/sync/work/devel/cli_weather/rhasspy_weather/data_types/report.py", line 163, in __generate_item_report
    if day_weather.is_clear:
AttributeError: 'NoneType' object has no attribute 'is_clear'
```

I am unsure what' the best way to fix this here. Maybe, we need to say time error or be explicit that this question does not make sense for today anymore. Or, no, you are fine, no sunscreen needed at night anymore - only if you are a vampire?

**Comment by Daenara** [comment]
There are answers for that scenario, if it crashes then I broke something during my big restructure. I will take care of it.

**Comment by Daenara** [comment]
fixed in 06f7bd6fdb53c17d2b42bb9493f9712336f7e2e8
